src/types/JsonSchemaObject.ts

```diff
@@ -0,0 +1,27 @@
+import { ajv } from '../ajv';
+import type { JSONSchemaType } from 'ajv';
+
+// The goal is to test an object is a valid JSON Schema (not against a schema, IS a schema).
+// This is a temporary interface. There is likely a better JSON Schema interface from ajv.
```

Comment:
In the interest of time and getting this change out for review, I didn't dig very deeply into Ajv or other libraries to find an existing JSON Schema, er, schema.

Reply:
I think instead of defining our own type here, we can use AJV's `JSONSchemaType`.